src/components/security_manager/src/ssl_context_impl.cc

```diff
@@ -136,32 +138,69 @@ std::map<std::string, CryptoManagerImpl::SSLContextImpl::BlockSizeGetter>
     CryptoManagerImpl::SSLContextImpl::max_block_sizes =
         CryptoManagerImpl::SSLContextImpl::create_max_block_sizes();
 
+std::string CryptoManagerImpl::SSLContextImpl::RemoveDisallowedInfo(
+    X509_NAME* in_data) const {
```

Comment:
@LuxoftAKutsan yes, it can. Added checking for `in_data` for this specific case in https://github.com/smartdevicelink/sdl_core/pull/1636/commits/299664d8d96664a1e3102685faa8fa9ef1de86c2